### ISSUE TITLE
please publish on PyPi

### ISSUE THREAD

**Issue by ilka-schulz** [Zahner-elektrik/Zahner-Analysis-Python#7]
I installed the package `thales_remote` which depends on this `zahner_analysis`. However, the latter one is not yet published on PyPi.

**Comment by maxkrapp1** [comment]
Could you please tell me more. on PyPi I have added both. The dependencies had actually also fit with me.

`pip install zahner_analysis` works.

https://pypi.org/project/zahner-analysis/
https://pypi.org/project/thales-remote/

**Comment by maxkrapp1** [comment]
I have freshly installed python 3.11 on a computer and tested the installation of the package.

```bash
C:\Users\John Doe>pip install thales_remote
Defaulting to user installation because normal site-packages is not writeable
Collecting thales_remote
  Using cached thales_remote-1.0.0-py3-none-any.whl (31 kB)
Collecting zahner-analysis
  Downloading zahner_analysis-1.0.2-py3-none-any.whl (30 kB)
Collecting zahner-potentiostat
  Downloading zahner_potentiostat-1.1.0-py3-none-any.whl (55 kB)
     ---------------------------------------- 56.0/56.0 kB 2.9 MB/s eta 0:00:00
Collecting numpy
  Downloading numpy-1.24.1-cp311-cp311-win_amd64.whl (14.8 MB)
     ---------------------------------------- 14.8/14.8 MB 5.8 MB/s eta 0:00:00
Collecting matplotlib
  Downloading matplotlib-3.6.3-cp311-cp311-win_amd64.whl (7.2 MB)
     ---------------------------------------- 7.2/7.2 MB 5.9 MB/s eta 0:00:00
Collecting requests
  Downloading requests-2.28.2-py3-none-any.whl (62 kB)
     ---------------------------------------- 62.8/62.8 kB 3.3 MB/s eta 0:00:00
Collecting pyserial
  Using cached pyserial-3.5-py2.py3-none-any.whl (90 kB)
Collecting contourpy>=1.0.1
  Downloading contourpy-1.0.7-cp311-cp311-win_amd64.whl (162 kB)
     ---------------------------------------- 163.0/163.0 kB 4.9 MB/s eta 0:00:00
Collecting cycler>=0.10
  Using cached cycler-0.11.0-py3-none-any.whl (6.4 kB)
Collecting fonttools>=4.22.0
  Downloading fonttools-4.38.0-py3-none-any.whl (965 kB)
     ---------------------------------------- 965.4/965.4 kB 5.1 MB/s eta 0:00:00
Collecting kiwisolver>=1.0.1
  Downloading kiwisolver-1.4.4-cp311-cp311-win_amd64.whl (55 kB)
     ---------------------------------------- 55.4/55.4 kB 3.0 MB/s eta 0:00:00
Collecting packaging>=20.0
  Downloading packaging-23.0-py3-none-any.whl (42 kB)
     ---------------------------------------- 42.7/42.7 kB ? eta 0:00:00
Collecting pillow>=6.2.0
  Downloading Pillow-9.4.0-cp311-cp311-win_amd64.whl (2.5 MB)
     ---------------------------------------- 2.5/2.5 MB 5.8 MB/s eta 0:00:00
Collecting pyparsing>=2.2.1
  Downloading pyparsing-3.0.9-py3-none-any.whl (98 kB)
     ---------------------------------------- 98.3/98.3 kB 5.9 MB/s eta 0:00:00
Collecting python-dateutil>=2.7
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting charset-normalizer<4,>=2
  Downloading charset_normalizer-3.0.1-cp311-cp311-win_amd64.whl (96 kB)
     ---------------------------------------- 96.0/96.0 kB 5.7 MB/s eta 0:00:00
Collecting idna<4,>=2.5
  Downloading idna-3.4-py3-none-any.whl (61 kB)
     ---------------------------------------- 61.5/61.5 kB 3.4 MB/s eta 0:00:00
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.14-py2.py3-none-any.whl (140 kB)
     ---------------------------------------- 140.6/140.6 kB 4.1 MB/s eta 0:00:00
Collecting certifi>=2017.4.17
  Downloading certifi-2022.12.7-py3-none-any.whl (155 kB)
     ---------------------------------------- 155.3/155.3 kB 4.7 MB/s eta 0:00:00
Collecting six>=1.5
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Installing collected packages: pyserial, charset-normalizer, urllib3, six, pyparsing, pillow, packaging, numpy, kiwisolver, idna, fonttools, cycler, certifi, requests, python-dateutil, contourpy, matplotlib, zahner-potentiostat, zahner-analysis, thales_remote
  WARNING: The scripts pyserial-miniterm.exe and pyserial-ports.exe are installed in 'C:\Users\John Doe\AppData\Roaming\Python\Python311\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script normalizer.exe is installed in 'C:\Users\John Doe\AppData\Roaming\Python\Python311\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script f2py.exe is installed in 'C:\Users\John Doe\AppData\Roaming\Python\Python311\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The scripts fonttools.exe, pyftmerge.exe, pyftsubset.exe and ttx.exe are installed in 'C:\Users\John Doe\AppData\Roaming\Python\Python311\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed certifi-2022.12.7 charset-normalizer-3.0.1 contourpy-1.0.7 cycler-0.11.0 fonttools-4.38.0 idna-3.4 kiwisolver-1.4.4 matplotlib-3.6.3 numpy-1.24.1 packaging-23.0 pillow-9.4.0 pyparsing-3.0.9 pyserial-3.5 python-dateutil-2.8.2 requests-2.28.2 six-1.16.0 thales_remote-1.0.0 urllib3-1.26.14 zahner-analysis-1.0.2 zahner-potentiostat-1.1.0

```

I think it was due to the python version greater than or equal to 3.10. I have now simply taken the 3.7 from you but not getestst. I chose 3.10 because I developed this with this version. I myself always develop with the latest python version.

**Comment by maxkrapp1** [comment]
I have published version 1.0.3 which allows the python version up to 3.7 according to #6 .

**Comment by maxkrapp1** [comment]
I have published the python package with 3.7 as requirement.
